misc/vasp/python/processors/Chgcar.py

```diff
@@ -0,0 +1,197 @@
+# Name: Chgcar
```

Comment:
Why capital C? (also in file name)

Reply:
Ok, commonly we append Source for source processors. Might want to do that here as well

Reply:
Occasionally, there is also `*Import` ;-)